generic-examples/http/files/public.cert

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
No, we should never put any keystore or certificates even if its a testing one. We can attract the interest of hacking maneuvers. Not sure if it was on purpose or they slipped into the commit by mistake.

Reply:
Done